src/main/api/2/studio-api-2.yaml

```diff
@@ -6920,6 +6920,307 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /workflow/packages:
+    get:
+      tags:
+        - workflow
+      summary: List workflow packages order by scheduled date
+      description: Required permission "requst_publish, publish"
+      operationId: workflowListPackage
+      parameters:
+        - name: siteId
+          in: query
+          description: filter workflow packages by site
+          required: true
+          schema:
+            type: string
+        - name: offset
+          in: query
+          description: Offset of the first package in the response
+          required: false
+          schema:
+            type: integer
+            format: int32
+        - name: limit
+          in: query
+          description: Number of packages to return
+          required: false
+          schema:
+            type: integer
+            format: int32
+        - name: order
+          in: query
+          description: ascending or descending (ASC | DESC). Default ascending
+          required: false
+          schema:
+            type: string       
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  total:
+                    type: integer
+                    format: int32
+                    description: Total records in the system
+                  offset:
+                    type: integer
+                    format: int32
+                    description: Offset of first record in this response
+                  limit:
+                    type: integer
+                    format: int32
+                    description: Number of records returned in this response
+                  packages:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/WorkflowPackage'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+    post:
+      tags:
+        - workflow
+      summary: Create workflow package
+      description: Required permission "requst_publish, publish"
+      operationId: workflowCreatePackage
+      requestBody:
+        description: parameters to create new workflow package
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId:
```

Comment:
Let's add a new property, `label`, to a package object. This will be the main thing the UI displays to be more user-friendly.